README.md

```diff
@@ -41,128 +41,70 @@ $ nigiri start --liquid
-* Go
-
-## Ingredients
-
-* [Bitcoin daemon](https://bitcoin.org/en/bitcoin-core/)
-* [Liquid daemon](https://blockstream.com/liquid/)
-* [Electrum server](https://github.com/Blockstream/electrs)
-* [Esplora](https://github.com/Blockstream/esplora)
-* [Nigiri Chopsticks](https://github.com/vulpemventures/nigiri-chopsticks)
-
-## Directions
-
-| Preparation Time: 5 min  | Cooking Difficulty: Easy |
-| --- | --- |
-
-* Clone the repo:
-
-```bash
-$ git clone https://github.com/vulpemventures/nigiri.git
-```
 
-* Enter project directory and install dependencies:
+**Note for users of macOS Monterey an onward**
+<details>
+  <summary>Show more...</summary>
+   When trying to start Nigiri, you might get an error similar to the following:
 
-```bash
-$ make install
-```
-
-
-* Build binary 
-
-```
-$ make build
-```
-
-Done! You should be able to find the binary in the local `./build` folder. Give it permission to execute and move/rename into your PATH.
-
-
-* Clean
-
-Remeber to always `clean` Nigiri before running `install` to upgrade to a new version.
-
-```
-$ make clean
-```
+  ```bash
+  Error response from daemon: Ports are not available: listen tcp 0.0.0.0:5000: bind: address already in use 
+  exit status 1
+  ```
+  This is due to AirPlay Receiver using port 5000, conflicting with Esplora trying to run using the very same port. 
 
+  There are two ways to deal with this issue:
 
+  1) Uncheck AirPlay Receiver in `System Preferences → Sharing → AirPlay Receiver`
+  2) Change Esplora’s port to something other than 5000. This can be done by changing it in [docker-compose.yml](https://github.com/vulpemventures/nigiri/blob/master/cmd/nigiri/resources/docker-compose.yml#L110) found in your data directory. If you previously tried starting Nigiri getting an error – you might have to run `nigiri stop --delete`  before restarting it.
+</details>
+<br />
 
 ## Tasting
 
 At the moment bitcoind, elements and electrs are started on *regtest* network.
 
 
-*  Start nigiri:
+### Start nigiri
 
 ```bash
 $ nigiri start
 ```
-Use the `--liquid` flag to let you do experiments with the Liquid sidechain. A liquid daemon and a block explorer are also started when passing this flag.
+- Use the `--liquid` flag to let you do experiments with the Liquid sidechain. A liquid daemon and a block explorer are also started when passing this flag.
+
+- Use the `--ln` flag to start a Core Lightning node and a LND node.
 
-* Stop nigiri:
+### Stop nigiri
 
 ```bash
 $ nigiri stop
 ```
 Use the `--delete` flag to not just stop Docker containers but also to remove them and delete the config file and any new data written in volumes.
 
-* Generate and send bitcoin to given address
+### Generate and send bitcoin to given address
 
 ```bash
 # Bitcoin
-$ nigiri faucet bcrt1qsl4j5je4gu3ecjle8lckl3u8yywh8rff6xxk2r
+$ nigiri faucet <bitcoin_address>
+## Fund the Core Lightning node
+$ nigiri faucet cln 0.01
+
+## Fund the LND node
+$ nigiri faucet lnd 0.01
```

Comment:
I'm following the readme like a noob and these 2 instructions are misleading.. what do you mean with `cln` and `lnd`? how to generate bitcoin addresses from those services?